express/features/support/steps/shared-functions.ts

```diff
@@ -18,11 +18,16 @@ export async function getButtonWithText(page: Page, buttonText: string): Promise
     return getSingleElement(buttons, buttonText);
 }
 
-export async function getButtonLink(page: Page, linkText: string): Promise<ElementHandle> {
+export async function getButtonAnchor(page: Page, linkText: string): Promise<ElementHandle> {
```

Comment:
Could we have added the fieldName to the component in the template instead of needing a separate function? (I confess to not being entirely sure what [@aria-controls="${fieldName}"] is actually selecting.)

Reply:
@aria-controls="${fieldName}" is providing a unique selection on the page as it has the field name that it was looking for on update-password page. It has two password inputs one with current-password and other one is password. Macros somehow not passing the name/id to uniquely identifying the elements. Do you want me to update the .njk?

Reply:
I see now aria-controls is an accessibility thing so keep the two different methods.

Reply:
There is a separate method getButtonLink() used for toggle specific steps only. I have renamed existing getButtonLink() to getButtonAnchor() to be more specific as it is searching the element with //a[contains(text()